### PR TITLE
Added styles for ActionBlock component

### LINKED ISSUE #2142: Add missing styles for `ActionBlock` component

### PR DESCRIPTION
**Pull request by vijay2136**
- Fixes #2142 

**Description**
Added: Styles for _ActionBlock_ wrapper

**Checklist**

- ~~[ ] I have made corresponding changes to the documentation.~~
- ~~[ ] I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- ~~[ ] I have added tests that prove my fix is effective or that my feature works.~~
- ~~[ ] I have added proper `data-cy` and `data-testid` attributes.~~
- [x] I have added the necessary label (`patch`/`minor`/`major` - If package publish
      is required).

**Reviewers**

### CODE REVIEW

**Comment by vijay2136** [comment]
@AbhayVAshokan _a Please take a look.

File: src/components/formik/ActionBlock.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const ActionBlock = ({
   const isSubmitting = isFormSubmitting ?? isFormikSubmitting;
 
   return (
-    <div className={classnames(["neeto-ui-action-block__wrapper", className])}>
+    <div className={classnames(["flex items-center gap-x-2", className])}>
```
Comment (AbhayVAshokan) [inline src/components/formik/ActionBlock.jsx:26]: @vijay2136 _a we do not use TailwindCSS within neetoUI. Please replace them with the corresponding [utility classes](https://github.com/bigbinary/neeto-ui/blob/main/src/styles/utilities/_utilities.scss) or SCSS.

**Comment by vijay2136** [comment]
@AbhayVAshokan _a Please take a look.

File: src/styles/components/_action-block.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.neeto-ui-action-block__wrapper {
+  display: flex;
+  align-items: center;
+  column-gap: 0.5rem;
```
Comment (praveen-murali-ind) [inline src/styles/components/_action-block.scss:4]: @vijay2136 _a Let’s use gap.
```suggestion
  gap: 0.5rem;
```

**Pull request merged**